apps/core/scenes/results/Results.js

```diff
@@ -84,6 +72,59 @@ export default class Results extends React.Component<Props> {
   }
 }
 
+const getSearchQueryByType = (type: TripTypes) => {
+  let query;
+  switch (type) {
+    case 'Return':
+      query = graphql`
+        query ResultsReturnQuery($input: ItinerariesReturnSearchInput!) {
+          searchData: searchReturnItineraries(input: $input) {
+            ...ResultsList_data
+          }
+        }
+      `;
+      break;
+    case 'OneWay':
+      query = graphql`
+        query ResultsOneWayQuery($input: ItinerariesOneWaySearchInput!) {
+          searchData: searchOneWayItineraries(input: $input) {
+            ...ResultsList_data
+          }
+        }
+      `;
+      break;
+  }
+  return query;
+};
+
+const parseSearchParametersByType = (
```

Comment:
Can you add this function inside the Component? The reason is consistency.

Reply:
Sure! Done